tensorflow_lite_support/cc/test/task/processor/BUILD

```diff
@@ -0,0 +1,45 @@
+load(
+    "@org_tensorflow//tensorflow/lite/core/shims:cc_library_with_tflite.bzl",
+    "cc_test_with_tflite",
+)
+
+package(
+    default_visibility = [
+        "//tensorflow_lite_support:users",
+    ],
+    licenses = ["notice"],  # Apache 2.0
+)
+
+cc_test_with_tflite(
+    name = "image_preprocessor_test",
+    srcs = ["image_preprocessor_test.cc"],
+    data = [
+        "//tensorflow_lite_support/cc/test/testdata/task/vision:test_images",
+        "//tensorflow_lite_support/cc/test/testdata/task/vision:test_models",
+    ],
+    tflite_deps = [
```

Comment:
A lot of the  tflite_deps and deps can be removed.

If a target is cc_library_with_tflite, use tflite_deps, otherwise, if it is cc_library, use deps. cc_library_with_tflite has not difference with cc_library in the github version, but Google internally, they will pull in different tflite version.